go.mod

```diff
@@ -12,6 +12,8 @@ require (
 	k8s.io/api v0.18.2
 	k8s.io/apimachinery v0.18.2
 	k8s.io/client-go v0.18.2
+	k8s.io/component-base v0.18.2
+	k8s.io/klog v1.0.0
```

Comment:
What requires these dependencies? Not an issue, but couldn't see where they were referenced.

Reply:
I couldn't see where either so I ran `go mod tidy` and it removed them. I must have deleted the code that used them!